test/tests/meshgenerators/move_node_generator/tests

```diff
@@ -0,0 +1,27 @@
+[Tests]
+  issues = '#18577'
+  design = 'meshgenerators/MoveNodeGenerator.md'
+
+  [generate]
+    requirement = 'The system shall support the ability to modify node(s) coordinates'
+
+    [single_element]
+      type = 'Exodiff'
+      input = 'test.i'
+      cli_args = '--mesh-only'
+      exodiff = 'test_in.e'
+      recover = false
+      mesh_mode = replicated
+
+      detail = 'in a mesh with a single first order element'
+
+    []
+    [3d]
+      type = 'Exodiff'
+      input = 'test2.i'
+      exodiff = 'test2_out.e'
+
+      detail = 'in a Cartesian 3D mesh.'
```

Comment:
Why does this test have a solve?

Reply:
@GiudGiud I'm honestly not sure why it was here again, could you expound on this?

Reply:
just to check that things still work after the mesh modification. 
I could go either way, but it was important for us to check that the solve works, because it does not if you modify the mesh too bad (inverted elements etc)